art/art_dic.py

```diff
@@ -84,6 +88,15 @@
     "cat face": "⦿⽘⦿",
     "cute cat": "^⨀ᴥ⨀^",
     "cute face": "(｡◕‿◕｡)",
+    "cute face2": "(ღ˘◡˘ღ)",
+    "cute face3": "✿◕ ‿ ◕✿",
+    "cute face4": "❀◕ ‿ ◕❀",
+    "cute face5": "(✿◠‿◠)",
+    "cute face6": "(◕‿◕✿)",
+    "cute face7": "☾˙❀‿❀˙☽",
+    "cute face8": "(◡‿◡✿)",
+    "cute face9": "◤(¬‿¬)◥",
```

Comment:
It doesn't seem so cute (it's more like octopus in sponge bob), can you change it to `upset` or something like this?

Reply:
Solved in https://github.com/sepandhaghighi/art/pull/141/commits/4c70fecfd0ba90d8a068bf5075a9354aeb78ee77